tests/functional/ext/private_import/private_import.txt

```diff
@@ -0,0 +1,11 @@
+import-private-name:5:0:5:24::Imported private module (_world):UNDEFINED
```

Comment:
I think we could pass `confidence=HIGH` to the `add_message`. Nothing in this checker is based on inference right? It's all just checking variable names.